pyprof/tests/L0/run_pyprof_nvtx/__init__.py

```diff
@@ -0,0 +1 @@
+import test_pyprof_nvtx.TestPyProfNvtx as TestPyProfNvtx
```

Comment:
if this change is unrelated to this PR, can it be removed and saved for a separate PR?

Reply:
It is already

Reply:
This change was due to the rebase.  It should be gone now.